tests/unit/docs/bcdoc/test_style.py

```diff
@@ -294,7 +293,7 @@ def test_write_py_doc_string(self):
             'returns: None'
         )
         style.write_py_doc_string(docstring)
-        self.assertEqual(style.doc.getvalue(), six.b(docstring + '\n'))
+        self.assertEqual(style.doc.getvalue(), docstring + b'\n')
```

Comment:
We need to make 291-293 bytes if you're going to do this.

```
        docstring = (
            b'This describes a function\n'
            b':param foo: Describes foo\n'
            b'returns: None'
        )
```

Reply:
I don’t understand, what am I supposed to do?

```python
stitny~$ python3.11
Python 3.11.0 (main, Oct 25 2022, 08:54:39) [GCC] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> docstring = (
...     b'This describes a function\n'
...     b':param foo: Describes foo\n'
...     b'returns: None'
... )
>>> len(docstring)
65
>>> type(docstring)
<class 'bytes'>
>>> 
```

Reply:
Tests are passing now. I guess, I got it, finally.